### DISCUSSION TITLE
gcc 编译不过看了一下 gcc要求这么高

### DISCUSSION THREAD

**Discussion by gooker** [alibaba/async_simple#293] [General]
gcc (>= 10.3) ,centos7 gcc devtoolset gcc是 10.2.1 差一点点

**Comment by ChuanqiXu9** [comment]
10.2.1 的 `<coroutine>` 头文件差了个 noexcept。你乐意改标准头文件的话，也可以绕过去的。

**Reply by gooker** [reply]
乐意改 能细说一下吗

**Reply by 4kangjc** [reply]
`coroutine_handle`的`resume`少了noexcept ?

**Reply by gooker** [reply]
> `coroutine_handle`的`resume`少了noexcept ?

是这个差异吧?

![image](https://user-images.githubusercontent.com/5688076/227183513-2eaabc77-a308-45b9-ab95-1dd006330b1c.png)

**Reply by gooker** [reply]
> `coroutine_handle`的`resume`少了noexcept ?

是这个差异吧?

![image](https://user-images.githubusercontent.com/5688076/227183513-2eaabc77-a308-45b9-ab95-1dd006330b1c.png)

**Reply by ChuanqiXu9** [reply]
是的，应该是这个。
